src/app/containers/FeaturesAnalysis/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, bool, node } from 'prop-types';
+import SectionLabel from '@bbc/psammead-section-label';
+import styled from 'styled-components';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+
+import featuresAnalysis from '#pages/StoryPage/featuresAnalysis.json';
+import { RequestContext } from '#contexts/RequestContext';
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { GhostGrid, GridItemConstrainedLarge } from '#lib/styledGrid';
+import StoryPromo from '../StoryPromo';
+import DivHeadline from '../StoryPromo/DivHeadlines';
+
+const Wrapper = styled(GridItemConstrainedLarge)`
+  margin-bottom: ${GEL_SPACING_DBL};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+  /* z-index needs explicitly set as the psammead section label component uses negative z-indices */
+  z-index: 0;
+`;
+
+const StyledSectionLabel = styled(SectionLabel)`
+  margin-top: 0;
+
+  span {
+    background-color: transparent;
+  }
+`;
+
+const FeaturesAnalysis = ({ content, enableGridWrapper }) => {
+  const { script, service, dir, translations } = useContext(ServiceContext);
+  const { env } = useContext(RequestContext);
+  const a11yAttributes = {
+    as: 'section',
+    role: 'region',
+    'aria-labelledby': 'features-analysis-heading',
+  };
+  const FeaturesAnalysisWrapper = ({ children }) =>
+    enableGridWrapper ? (
+      <GhostGrid {...a11yAttributes}>
+        <Wrapper>{children}</Wrapper>
+      </GhostGrid>
+    ) : (
+      <Wrapper {...a11yAttributes}>{children}</Wrapper>
+    );
+  FeaturesAnalysisWrapper.propTypes = {
+    children: node.isRequired,
+  };
+  if (!featuresAnalysis.length) return null;
+
+  return (
+    <FeaturesAnalysisWrapper>
+      <Wrapper>
+        <StyledSectionLabel
+          script={script}
+          service={service}
+          dir={dir}
+          labelId="features-analysis-heading"
+        >
+          Featured &amp; Analysis {/** this should be translated */}
```

Comment:
```suggestion
          Features &amp; Analysis {/** this should be translated */}
```
Also are the translations not part of this PR?

Reply:
Note: Check with @lizcameron

Reply:
This is being tracked in https://github.com/bbc/simorgh/issues/5794, which I have assigned to myself, and intend to complete in conjunction with https://github.com/bbc/simorgh/issues/5765

Reply:
Thanks for addressing the translation point, but placeholder text still has a typo we can correct for now 🙂